serializable.ts

```diff
@@ -103,6 +107,15 @@ export abstract class Serializable {
   public tsSerialize(): JSONObject {
     return toPojo(this);
   }
+  public clone(jsonObject: Partial<this> = {}): this {
```

Comment:
I feel that we need a JSDoc comment here explaining what the argument does. It may not be obvious to users who use ts_serialize differently than we do.

Reply:
added on the interrface